stateful-workload-filestore/indexInfo.html

```diff
@@ -22,22 +22,21 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
 
     <script>
-        $(function () {
```

Comment:
Change 3: I used the [Google JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html) to make slight tweaks to the style.